### PR TITLE
Adding changes generated from openshift/knative-eventing#949

### PR DESCRIPTION
**Pull request by cardil**
As title says: "Adding changes generated from openshift/knative-eventing#949"

### CODE REVIEW

File: ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.17.2__45.yaml [hunk 1]
```diff
@@ -0,0 +1,280 @@
+base_images:
```
Comment (matzew) [inline ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.17.2__45.yaml:1]: @cardil  pleaes remove this file.

We have disable the 4.5 run for the 0.17+ branches.

File: ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.17.2__46.yaml [hunk 2]
```diff
@@ -236,6 +244,7 @@ images:
 promotion:
   additional_images:
     knative-eventing-src: src
+  disabled: true
```
Comment (matzew) [inline ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.17.2__46.yaml:247]: since we only have 4.6, remove _this_ line

**Comment by matzew** [comment]
@cardil  as said:

* remove the one file for 4.5
* remove the one disabled line for the promotion

Once that is done, on your `openshift/release` branch's root, run ` make jobs && make ci-operator-config`

File: ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.17.2__45.yaml (hunk 1 above)
Comment (cardil) [reply ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.17.2__45.yaml:1]: The script generated it. Should we update the script?

Comment (cardil) [reply ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.17.2__45.yaml:1]: https://github.com/openshift/knative-eventing/pull/950 should do the trick

**Comment by cardil** [comment]
/hold

openshift/knative-eventing#949 needs to be merged first

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Comment by matzew** [comment]
/hold

:-) 

let's see if th eimage build is green

**Pull request merged**